bindings_ffi/src/mls.rs

```diff
@@ -647,6 +651,7 @@ impl FfiCreateGroupOptions {
             name: self.group_name,
             image_url_square: self.group_image_url_square,
             description: self.group_description,
+            pinned_frame: self.pinned_frame,
```

Comment:
Should this be pinned frame url to be more explicit it's a url?

Reply:
probably right I'll update that and fix these node bindings tests before merging 👌

Reply:
updated https://github.com/xmtp/libxmtp/pull/888/commits/656c2c69ae27b20a7f07f13e0e69d9c13563bd8a